testing/trino-benchto-benchmarks/README.md

```diff
@@ -18,7 +18,8 @@ features. Therefore benchmarks are organized in suites, like:
 
 ### Requirements
 
-* Presto already installed on the target environment
+* JDK 8 for compiling and running Benchto
+* Trino already installed on the target environment
```

Comment:
This shouldn't be part of `Pin java version for trino-benchto-benchmarks module to JDK 8` commit